e2e/test/csi_driver_test.go

```diff
@@ -61,33 +61,71 @@ var _ = Describe("Linode CSI Driver", func() {
 
 	Describe("Test", func() {
 		Context("Simple", func() {
+			var (
+				f    *framework.Invocation
```

Comment:
Can/should we share some (all?) of these atop of this group of tests?

Reply:
someone decided these needed to be scoped to the Describe... i dont know if i can properly answer this question yet but I agree with you and ill give it a try and see where it comes out